--- src/python/grpcio/commands.py	2020-06-22 17:42:27.000000000 -0400
+++ src/python/grpcio/commands.py	2020-12-03 08:24:28.937322206 -0500
@@ -217,6 +217,8 @@
             """Test if default compiler is okay with specifying c++ version
             when invoked in C mode. GCC is okay with this, while clang is not.
             """
+            if sys.platform == "win32":
+                return False
             try:
                 # TODO(lidiz) Remove the generated a.out for success tests.
                 cc_test = subprocess.Popen(['cc', '-x', 'c', '-std=c++11', '-'],
