From b4d397a7886d6115ddb91431fc966ea0f17fac48 Mon Sep 17 00:00:00 2001
From: Silvio Traversaro <silvio@traversaro.it>
Date: Sun, 11 Apr 2021 15:41:48 +0200
Subject: [PATCH] Disable GUI

---
 dart/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/dart/CMakeLists.txt b/dart/CMakeLists.txt
index c3ebbdc04d6..1f4d15273f1 100644
--- a/dart/CMakeLists.txt
+++ b/dart/CMakeLists.txt
@@ -86,7 +86,7 @@ add_subdirectory(constraint)
 add_subdirectory(simulation)
 add_subdirectory(planning) # flann
 add_subdirectory(utils) # tinyxml2, bullet
-add_subdirectory(gui) # opengl, glut, bullet
+# add_subdirectory(gui) # opengl, glut, bullet
 
 set(DART_CONFIG_HPP_IN ${DART_SOURCE_DIR}/dart/config.hpp.in)
 set(DART_CONFIG_HPP_OUT ${DART_BINARY_DIR}/dart/config.hpp)
 
